statsforecast/core.py

```diff
@@ -1546,6 +1551,113 @@ def plot(
             palette="tab20b",
         )
 
+    def save(
+        self,
+        path: Union[os.PathLike, str],
```

Comment:
This doesn't seem to support pathlike objects, we can either just accept strings here or use [pathlib](https://docs.python.org/3/library/pathlib.html) inside the method